src/model/resources/curtailable_variable_renewable/curtailable_variable_renewable.jl

```diff
@@ -77,11 +79,10 @@ function curtailable_variable_renewable!(EP::Model, inputs::Dict, setup::Dict)
     for y in VRE_NO_POWER_OUT
         fix.(EP[:vP][y, :], 0.0, force = true)
     end
-    ##CO2 Polcy Module VRE Generation by zone
-    @expression(EP, eGenerationByVRE[z = 1:Z, t = 1:T], # the unit is GW
-        sum(EP[:vP][y, t]
-        for y in intersect(inputs["VRE"], resources_in_zone_by_rid(gen, z))))
-    add_similar_to_expression!(EP[:eGenerationByZone], eGenerationByVRE)
+    ##CO2 Policy Module VRE Generation by zone
+    # We use the transpose here because eGenerationByZone is [1:Z, 1:T] and
+    # eGenerationByVRE is [1:T, 1:Z].
+    add_similar_to_expression!(EP[:eGenerationByZone], eGenerationByVRE')
```

Comment:
Unless I've missed something, `eGenerationByVRE` is just the transpose of `ePowerBalanceDisp` above?

Reply:
Yes, you're right, the two expressions were just transposed versions of each other. Thanks for pointing this out.